frontend/catalyst/pennylane_extensions.py

```diff
@@ -1734,6 +1735,47 @@ def circuit(x: float):
     return out_classical_tracer
 
 
+# Note that "print" would alias the built-in print function.
+def debug_print(x):
+    """A :func:`qjit` compatible print function for debugging.
```

Comment:
In a sense, `print` is _also_ `qjit` compatible, it just won't behave like a user expects -- rather than erroring, it will print out at tracing/compile time rather than at runtime.

Perhaps it's more accurate to say here
```suggestion
    """A :func:`qjit` compatible print function for printing values during compiled function runtime.
```

Reply:
Good point 👍

Reply:
I will say that the say could be said about `for` and many other Python statements, they all work (to some degree) at compile time 😅 I guess this is just our standard header, but your suggestion makes sense.

Reply:
```suggestion
    """A :func:`qjit` compatible print function for printing values at runtime.
```